arch/riscv/mach-duowen/Kconfig

```diff
@@ -645,6 +645,10 @@ config DUOWEN_SSI_FLASH_IRQ
 
 endif
 
+config DUOWEN_SSI_FLASH_DTS
+	bool "Enable dw-apb-ssi driver in DTS"
+	depends FDT
+
 menuconfig DUOWEN_SD
```

Comment:
Should it be DUOWEN_SSI_MASTER_DTS, and be put between if DUOWEN_SSI_FLASH/endif?

Reply:
done